src/ejabberd_sm_redis.erl

```diff
@@ -75,9 +76,9 @@ get_sessions(User, Server, Resource) ->
 
     lists:map(fun(S) -> binary_to_term(S) end, Sessions).
 
--spec create_session(User :: jid:user(),
-                     Server :: jid:server(),
-                     Resource :: jid:resource(),
+-spec create_session(User :: jid:luser(),
+                     Server :: jid:lserver(),
+                     Resource :: jid:lresource(),
```

Comment:
Here there's an explicit change to the lowecase version, but in `ejabberd_sm_mnesia` there's not, so it seems inconsistent :slightly_frowning_face:

Reply:
Right, I forgot about `_mnesia`. Good catch.

Reply:
Now `_mnesia` is changed as well.